lib/tasks/docker.rake

```diff
@@ -57,15 +62,7 @@ namespace :docker do
   end
 
   task run_ci: :environment do
-    print `docker run \
-            -d=true \
-            --name felix \
-            -it \
-            -e SOLR_HEAP=1G \
-            -p 8983:8983 \
-            -v "$(pwd)"/solr/conf:/myconfig \
-            solr:7.4.0 \
-            -DzkRun`
+    print `docker run -d -p 8983:8983 solr:7.4.0 -DzkRun`
```

Comment:
@cdmo no need for `-e` anymore 👍

Reply:
And removed the mounting of the config because that's done via SolrManager now. Also removed the name on the instance.